src/components/actions/Button/Button.module.css

```
@@ -0,0 +1,185 @@
+  border-radius: 50%;
+  animation: zp-btn-spin 600ms linear infinite;
+}
+
+.contentHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)